server/apps/publish/archive_publish.py

```diff
@@ -188,7 +188,7 @@ def queue_transmission(self, doc, target_output_channels=None):
 
                 destination_groups = self.resolve_destination_groups(
                     doc.get('destination_groups'))
-                output_channels, selector_codes, format_types = \
+                output_channels, selector_codes, _format_types = \
```

Comment:
Hey Mugur, while I was looking at this change that I noticed an issue here, though it is nothing to do with your changes. We iterate through every output channel and format the story per output channel. We may have more than one output channel having the same format so we don't need to re-format again. I have created a technical task https://dev.sourcefabric.org/browse/SD-2420. So again thanks to your change in that field name.